java.lang.IndexOutOfBoundsException: 0
occurred in the presentation compiler.

action parameters:
offset: 287
uri: file://<WORKSPACE>/src/test/scala/V_Single-Cycle/VECTOR_REGISTER/VECTOR_REGISTERTest.scala
text:
package Vpractice

import chisel3._
import org.scalatest.FreeSpec
import chiseltest._

class VECTOR_REGISTERTest extends FreeSpec with ChiselScalatestTester{
   "V_REGISTERTest test" in{
       test(new V_RegFile ()){c =>
         c.io.reg_sew.poke("b010".U)
         c.io.reg_lmul.poke(@@)
         c.io.vs1.poke(1.U)
         c.io.vs2.poke(3.U)
         c.io.V_reg_write.poke(1.B)
         c.io.vd.poke(1.U)
         c.io.V_w_data(0).poke(8.U) // Writing sequential data
         c.io.V_w_data(1).poke(8.U) // Writing sequential data
         c.io.V_w_data(2).poke(8.U) // Writing sequential data
         c.io.V_w_data(3).poke(8.U) // Writing sequential data
         c.io.V_w_data(4).poke(8.U)
         c.clock.step(50)
      }
   }
   }



error stacktrace:
scala.collection.LinearSeqOps.apply(LinearSeq.scala:131)
	scala.collection.LinearSeqOps.apply$(LinearSeq.scala:128)
	scala.collection.immutable.List.apply(List.scala:79)
	dotty.tools.dotc.util.Signatures$.countParams(Signatures.scala:501)
	dotty.tools.dotc.util.Signatures$.applyCallInfo(Signatures.scala:186)
	dotty.tools.dotc.util.Signatures$.computeSignatureHelp(Signatures.scala:94)
	dotty.tools.dotc.util.Signatures$.signatureHelp(Signatures.scala:63)
	scala.meta.internal.pc.MetalsSignatures$.signatures(MetalsSignatures.scala:17)
	scala.meta.internal.pc.SignatureHelpProvider$.signatureHelp(SignatureHelpProvider.scala:51)
	scala.meta.internal.pc.ScalaPresentationCompiler.signatureHelp$$anonfun$1(ScalaPresentationCompiler.scala:371)
